src/css/support.js

```diff
@@ -54,9 +54,9 @@ support.reliableTrDimensions = function() {
 		}
 
 		trStyle = window.getComputedStyle( tr );
-		reliableTrDimensionsVal = ( parseInt( trStyle.height, 10 ) +
-				parseInt( trStyle.borderTopWidth, 10 ) +
-				parseInt( trStyle.borderBottomWidth, 10 ) ) === tr.offsetHeight;
+		reliableTrDimensionsVal = ( Math.round( parseFloat( trStyle.height ) ) +
+			Math.round( parseFloat( trStyle.borderTopWidth ) ) +
+			Math.round( parseFloat( trStyle.borderBottomWidth ) ) ) === tr.offsetHeight;
```

Comment:
IE / Edge, for which this support test was written, have much bigger differences here (something like 3 vs. 9) so rounding shouldn't affect it.